### ISSUE TITLE
Global permissions on UI are in a random order

### ISSUE THREAD

**Issue by anupama2501** [rancher/dashboard#11010]
**Setup**
- Rancher version: v2.7.13-rc4
- Rancher UI Extensions: `release-2.7 d14964b0d`
- Browser type & version: Chrome

**Describe the bug**
Global permissions on the Users and authentication page, have the permissions in a different order. 

**To Reproduce**
1. Install rancher 2.7.13-rc4
2. Navigate to users and authentication >> Users 
3. Create a user
4. Notice the order of the global permissions

**Result**
```
Restricted Admin
User base
Administrator
Standard user
```

**Expected Result**
Expected the order to be:
```
Administrator
Restricted Admin
Standard user
User base
```

**Screenshots**
<img width="1171" alt="2024-05-10_16-43-26" src="https://github.com/rancher/dashboard/assets/38144301/6293e3d3-7187-4c85-a09f-8e708cad2e53">

**Additional context**:
Do not see the issue on v2.7.12

**Comment by richard-cox** [comment]
I see a slightly different order in 2.9-head that is probably the same issue and should be fixed

1. Admin
2. User-Base
3. Restricted Administrator 
4. Standard User

**Comment by gaktive** [comment]
For docs (cc: @btat) we need a release note for the current 2.7.next1 (2.7.13) as we move this work to a later version. It is a regression though the UI around this hasn't changed in years so we'll have to look at the data given to us.

**Comment by gaktive** [comment]
Will not fix for 2.7.x. We have tickets for 2.9.x & 2.8.x at this stage though based on timing, we have to sort this out in 2.9.x first (or even 2.10.0 now)
